pkgs/servers/echoip/default.nix

```diff
@@ -1,21 +1,17 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   name = "echoip-${version}";
-  version = "unstable-2018-11-20";
-
-  goPackagePath = "github.com/mpolden/echoip";
+  version = "unstable-2019-04-27";
 
   src = fetchFromGitHub {
     owner = "mpolden";
     repo = "echoip";
-    rev = "4bfaf671b9f75a7b2b37543b2991401cbf57f1f0";
-    sha256 = "0n5d9i8cc5lqgy5apqd3zhyl3h1xjacf612z8xpvbm75jnllcvxy";
+    rev = "27fa828efb9034f359f638d29eda1f84aa08cc48";
+    sha256 = "1rp2vgpfc514qgq715rkhhiciz6k8ms19qs31q1an3aapacf446p";
   };
 
-  goDeps = ./deps.nix;
-
-  outputs = [ "bin" "out" ];
+  modSha256 = "025p891klwpid5fw4z39fimgfkwgkcwqpn5276hflzdp1hfv35ly";
 
   postInstall = ''
```

Comment:
Things with `buildGoModule` only have one output `out`.

So you just need to find an appropriate place for this within FHS, or maybe add another output for it.

Reply:
What is the issue here?
The only reason there was `out` output defined was to put `index.html` there. This PR does not modify that behavior.

Reply:
Is the index file required for runtime, or is this more of a documentation?

Reply:
It is required (though using an alternate `index.html` is supported) at runtime. Without a functioning front-end running the server is useless.
```
nix run nixpkgs.echoip -c echoip
echoip: Not configuring default handler: Template not found: index.html
echoip: Listening on http://0.0.0.0:8080
```

Reply:
Example usage can be seen here: https://github.com/rvolosatovs/infrastructure/blob/ccd0223e732716bc17b2bafb1bfa611305b78e9d/nixos/echoip.nix#L16

Reply:
I see you can specify it with the `-t` flag. Is it possible for it load it from a default location?

Also the issue here was just that `out` already exists since things using `buildGoModule` only have one output `out`. Whereas before with `buildGoPackage` it was `bin` and then `out`.

So maybe just add another output `index` for this file.

Reply:
The change you propose would be non-backwards compatible.
Setups depending on `${pkgs.echoip.out}/index.html` would break.
Also it seems there's not a single other derivation with output `index` defined.
Is there a more standardized naming scheme for things like this?